index.html

```diff
@@ -0,0 +1,205 @@
+              <a href="#">Contact</a>
+            </li>
+            <li>
+              <a href="#" class="news">News</a>
+            </li>
+          </ul>
+        </section>
+      </nav>
+    </header>
+    <main>
+      <section class="main-sec">
+        <div class="title-sec">
+          <h4 class="orange">FEDERAL MINISTRY OF</h4>
+          <h1 class="orange">COMMUNICATIONS AND DIGITAL ECONOMY</h1>
+        </div>
+        <section class="mission-vision">
+          <h3>Vision</h3>
+          <p>
+            To create a robust digital economy that enhances security, increases 
+            transparency, and diversifies the Nigerian economy.
+          </p>
+          <h3>Mission</h3>
+          <p>We leverage Digital Technologies for National Economic Development.</p>
+        </section>
+      </section>
+      <section class="program">
+        <section class="sec-tit">
+          <h2 class="gray">Our Agencies</h2>
+          <hr class="hr">
+        </section>
+        <ul class="agencies">
+          <li class="agency1">
+            <img src="images/ncc.png" alt="NCC">
+            <h3 class="orange">NCC</h3>
+            <p>
+              Provides exposure to the cadets in a wide range of activities., with a distinct 
+              emphasis on Social Services, Discipline and Adventure Training.
+            </p>
+          </li>
+          <li class="agency1">
+            <img src="images/NigComSat.png" alt="NigComSat">
+            <h3 class="orange">NigCom<br>Sat</h3>
+            <p>
+              Leading communication satellite operator and service 
+              provider in Africa.
+            </p>
+          </li>
+          <li class="agency1">
+            <img src="images/nimc.png" alt="nimc">
+            <h3 class="orange">NIMC</h3>
+            <p>
+              Operate, maintain and manage the National Identity Database in Nigeria, 
+            </p>
+          </li>
+          <li class="agency1">
+            <img src="images/nitda.png" alt="nitda">
+            <h3 class="orange">NITDA</h3>
+            <p>
+              Our mandate is to create a framework for the planning, research, development, 
+              standardization, application, coordination, monitoring, evaluation and regulation 
+              of Information Technology practices in Nigeria.
+            </p>
+          </li>
+          <li class="agency1">
+            <img src="images/nipost.jpg" alt="galaxy backbone">
+            <h3 class="orange">NIPOST</h3>
+            <p>
+              Is the Nigerian postal administration responsible for 
+              providing postal services in Nigeria.
+            </p>
+          </li>
+        </ul>
+        <section class="contact">
+          <a href="#">See Our Agencies</a>
+        </section>
+      </section>
+      <section class="leadership">
+        <h3>Leadership & Directors</h3>
+        <hr class="hr">
+        <section class="directors-list">
+          
+        </section>
```

Comment:
- [ ] Kindly add this button to the leadership and directors section in the mobile version of your design to remain consistent with the original design

![image](https://user-images.githubusercontent.com/98453979/235204683-d26176f6-9482-40a1-92ca-0c6d66276d2f.png)

Reply:
@frank1738 Thank you for the review, but this more button is not in the requirements of the capstone project, I have seen many students that did the work without it, and their work was accepted please reconsider this review.